test/bundle-test-config.yaml

```diff
@@ -7,6 +7,10 @@ defaults:
     organizationRoleName: none
     revolverRoleName: none
     localResourcesFile: bundle-test-resources.json
+    audit:
+      console:
+    excludeResources:
+      - tag: 'Schedule|0x7'
```

Comment:
Can you change this to `Schedule|ignore`?

0x7 has a real meaning, and resources will be assertively shutdown, not just ignored.